plugins/resources/app/controllers/resources/application_controller.rb

```diff
@@ -16,22 +16,18 @@ def bigvm_resources
       # https://migration-recommender-service.cca-pro.cerebro.c.eu-de-1.cloud.sap/public/docs#/default/get_placeable_vm_for_project_api_v1_placeable_vm_project__openstack_project_id__get
       require "net/http"
       begin
-        cerebro_endpoint = "https://migration-recommender-service.cca-pro.cerebro.c.#{current_region}.cloud.sap/public/api/v1/placeable-vm/#{openstack_level}"
+        cerebro_endpoint = "https://migration-recommender-service.#{current_region}.cloud.sap/public/api/v1/placeable-vm/#{openstack_level}"
         if ENV.key?("CEREBRO_CUSTOM_ENDPOINT") 
           unless ENV["CEREBRO_CUSTOM_ENDPOINT"].empty? || ENV["CEREBRO_CUSTOM_ENDPOINT"].blank?
             cerebro_endpoint = "#{ENV['CEREBRO_CUSTOM_ENDPOINT']}/public/api/v1/placeable-vm/#{openstack_level}"
```

Comment:
Do we really need to check if it is empty or blank?

I think blank already includes empty. 

Something like this maybe? Not tested!

```rails

cerebro_endpoint = if ENV["CEREBRO_CUSTOM_ENDPOINT"].present?
  "#{ENV['CEREBRO_CUSTOM_ENDPOINT']}/public/api/v1/placeable-vm/#{openstack_level}"
else
  "https://migration-recommender-service.#{current_region}.cloud.sap/public/api/v1/placeable-vm/#{openstack_level}"
end

```

Reply:
yup that could be a more simple way, will try